Duckling/Numeral/HI/Rules.hs

```diff
@@ -0,0 +1,181 @@
+-- Copyright (c) 2016-present, Facebook, Inc.
+-- All rights reserved.
+--
+-- This source code is licensed under the BSD-style license found in the
+-- LICENSE file in the root directory of this source tree. An additional grant
+-- of patent rights can be found in the PATENTS file in the same directory.
+
+-------------------------WORK IN PROGRESS------------------------
+
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE NoRebindableSyntax #-}
+
+module Duckling.Numeral.HI.Rules
+  ( rules
+  ) where
+
+import Data.HashMap.Strict (HashMap)
+import Data.Maybe
+import Data.String
+import Data.Text (Text)
+import Prelude
+import qualified Data.HashMap.Strict as HashMap
+import qualified Data.Text as Text
+
+import Duckling.Dimensions.Types
+import Duckling.Numeral.Helpers
+import Duckling.Numeral.Types (NumeralData (..))
+import Duckling.Regex.Types
+import Duckling.Types
+import qualified Duckling.Numeral.Types as TNumeral
+
+ruleIntegerNumeric :: Rule
+ruleIntegerNumeric = Rule
+  { name = "integer (numeric)"
+  , pattern =
+    [ regex "(\\d{1,18})"
+    ]
+  , prod = \tokens -> case tokens of
+      (Token RegexMatch (GroupMatch (match:_)):
+       _) -> do
+         v <- parseInt match
+         integer $ toInteger v
+      _ -> Nothing
+  }
+
+ruleNumeralMap :: HashMap Text Integer
+ruleNumeralMap = HashMap.fromList
+  [ ( "शून्य", 0)
+--  , ( "z\x00E9r\x00F3", 0 )
+  , ( "एक", 1 )
+--  , ( "kett\x0151",  2 )
+  , ( "दो", 2)
+--  , ( "h\x00E1rom", 3 )
+  , ( "तीन", 3 )
+--  , ( "n\x00E9gy", 4 )
+  , ( "चार", 4)
+--  , ( "\x00F6t", 5)
+  , ( "पाँच",5)
+--  , ( "hat", 6)
+  , ( "छह", 6)
+--  , ( "h\x00E9t", 7)
+  , ( "सात", 7)
+--  , ( "nyolc", 8)
+  , ( "आठ", 8)
+--  , ( "kilenc", 9)
+  , ("नौ", 9)
+--  , ( "t\x00EDz", 10)
+  , ("दस", 10)
+  ]
+
+ruleNumeral :: Rule
+ruleNumeral = Rule
+  { name = "number (0..10)"
+  , pattern =
+    [ regex "(शून्य|एक|दो|तीन|चार|पाँच|छह|सात|आठ|नौ|दस)"
```

Comment:
seems like this regex is missing western strings that are present in `ruleNumeralMap`